### ISSUE TITLE
[Bug]:  Germany > lrasha_de Waste Collection Integration Broken: ICS Source URL changed

### ISSUE THREAD

**Issue by RedDaveNY** [mampfes/hacs_waste_collection_schedule#3350]
### I Have A Problem With:

A specific source

### What's Your Problem

The waste collection integration for Landkreis Schwäbisch Hall (SHA) is no longer functional because **the ICS file source previously used is no longer available**. The SHA now provides waste collection schedules only as individual PDFs per route or through their dedicated Android/iOS app.

The new URL ( i was able to export the link inside the app) webcal://api.cross-7.de/public/calendar/399/events/ics?CategoryIds=68352&CategoryIds=68354&CategoryIds=68353&SecondCategoryIds=68313

The number after calendar ist now 399, not 396 anymore
I changed the python file on my HA and was able to integrate the calendar

### Source (if relevant)

https://www.lrasha.de/abfall/muellkalender/abfuhrtermine

### Logs

```Shell
The source returned an invalid response: "500 Server Error: Internal Server Error for url: https://api.cross-7.de/public/calendar/396/events/ics?SecondCategoryIds=68345". Please check the provided arguments and try again.
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: lrasha_de
      args:
        location: "68345"
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by RedDaveNY** [comment]
the lrasha_DE.py python file in my HA had still the wrong url.
After I changed it manual it works now fine.

Dont really know if this is an issue or unclean update with my fault

**Comment by DerDreschner** [comment]
Was fixed/changed in #3205. Just to keep everything linked together.
